chains/btc/config/config.go

```diff
@@ -32,18 +32,23 @@ type Resource struct {
 	Script     []byte
 }
 
+type UploaderConfig struct {
+	URL   string `mapstructure:"url" default:"https://api.pinata.cloud/pinning/pinFileToIPFS"`
+	Token string `mapstructure:"token" default:"eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJ1c2VySW5mb3JtYXRpb24iOnsiaWQiOiJkOTU2YTZiNC1mOWFlLTRmZDctYTc5My0zZTFmM2FjYzc4MjIiLCJlbWFpbCI6InRjYXIxMjEyOTNAZ21haWwuY29tIiwiZW1haWxfdmVyaWZpZWQiOnRydWUsInBpbl9wb2xpY3kiOnsicmVnaW9ucyI6W3siZGVzaXJlZFJlcGxpY2F0aW9uQ291bnQiOjEsImlkIjoiRlJBMSJ9LHsiZGVzaXJlZFJlcGxpY2F0aW9uQ291bnQiOjEsImlkIjoiTllDMSJ9XSwidmVyc2lvbiI6MX0sIm1mYV9lbmFibGVkIjpmYWxzZSwic3RhdHVzIjoiQUNUSVZFIn0sImF1dGhlbnRpY2F0aW9uVHlwZSI6InNjb3BlZEtleSIsInNjb3BlZEtleUtleSI6ImNkZDljZGFmYWJhOTYxMGYxNDdjIiwic2NvcGVkS2V5U2VjcmV0IjoiMGI0OTQ4NmZjMTYzNmQxZTIyZTE5OGFlZWViOGE2NmRjMDM4NmRmNWRlOTkyY2NlOTlkZDBiYzQ5YzBiODdmYSIsImV4cCI6MTc1MzM2MTkxN30.xP92Ts9661M3CjxIEKobZ_jwdIPmxb_m2POJ4mpNBss"`
```

Comment:
I would prefer not to have the token here. 
If we are gonna have a default gateway it should be some open one. 

This key you can basically throw away now.